front/lib/dust_api.ts

```diff
@@ -0,0 +1,69 @@
+import { Project } from "@app/types/project";
+
+const { DUST_API: DUST_API_URL } = process.env;
+
+type ErrorRsponse = { error: string };
```

Comment:
We return error objects not strings from core?

Reply:
Ah my bad